docs/payload.md

```diff
@@ -0,0 +1,113 @@
+        "destination_domain": 0,
+        "mint_recipient": "PNWAxASH2RPmgMV+/Tb4e78ON1WL8SoFGnwbWWHxfuA=",
+        "destination_caller": "xWtN0TuqjWo90XiknI61JUxYexN2JgZaEaWGxhA/rXE="
+      },
+      "passthrough_payload": ""
+    }
+  }
+}
+```
+
+## Actions
+
+An
+[`Action`](https://github.com/noble-assets/orbiter/blob/main/proto/noble/orbiter/core/v1/orbiter.proto#L12-L33)
+is a state transition request which can be executed on Noble and is defined by two fields:
+
+1. An action
+   [identifier](https://github.com/noble-assets/orbiter/blob/main/proto/noble/orbiter/core/v1/id.proto#L9-L24).
+2. An attributes field containing all the information to execute the action.
+
+The attributes field is defined as an `Any` type implementing the
+[`ActionAttributes`](https://github.com/noble-assets/orbiter/blob/main/types/core/attributes.go#L27-L31)
+
+Actions are executed in the order in which they are passed into the field `pre_actions`, and each of
+them operates on the state resulting from the execution of the previous one. For example, if the pre
+actions field is composed of a **swap** and a **fee** payment, the fee payment will be applied on
+the amount and denomination of the coin resulting from the swap.
+
+If any of the specified action fails, the entire state transition defined by the execution of the
+transaction will be reverted.
+
+### Fee
+
+Is it possible to specify fee payments as a single action by using the
+[`FeeAttributes`](https://github.com/noble-assets/orbiter/blob/main/types/controller/action/fee.pb.go#L26-L31).
+A single fee is defined by:
+
+- `Recipient`: A Noble address which will receive the fee.
+- `BasisPoints (BPS)`: A number between 0 and 10000 which defines the percentage of the transferred
+  amount that has to be paid as a fee. The fee amount will be defined as
+  $fee = amount \cdot \frac{BPS}{10000}$
+
+## Forwarding
+
+A
+[`Forwarding`](https://github.com/noble-assets/orbiter/blob/main/proto/noble/orbiter/core/v1/orbiter.proto#L35-L55)
+is an operation defined as a combination of:
+
+1. A bridge protocol
+   [identifier](https://github.com/noble-assets/orbiter/blob/main/proto/noble/orbiter/core/v1/id.proto#L26-L44).
+2. Attributes required to operate via the bridge protocol.
+3. A pass-through payload to forward with the outgoing transfer protocol.
+
+The attributes field is defined as an `Any` type implementing the
+[`ForwardingAttributes`](https://github.com/noble-assets/orbiter/blob/main/types/core/attributes.go#L33-L39)
+
+Based on the bridge protocol, the following conditions may result from the execution of an orbit:
+
+- Synchronous protocols: Protocols like CCTP and Hyperlane follow a commit-and-forget style. In this
+  case, once the associated server informs the Orbiter that the bridge request has been stored to
+  state, the Orbiter execution is complete. If the server returns an error, the entire tx will be
+  marked as unsuccessful.
```

Comment:
nit: haven't used abbreviations in the rest of the doc so we shouldn't start here


```suggestion
  state, the Orbiter execution is complete. If the server returns an error, the entire transaction will be
  marked as unsuccessful.
```